src/features/dashboard/components/Dialogs/DeleteAppDialog/delete-app-dialog.scss

```diff
@@ -0,0 +1,44 @@
+.modal-wrap {
+  display: flex;
```

Comment:
```  display: flex;
  justify-content: center;
  align-items: center; ```
  
  This seems like used mutliple times... why not creating a reusable class? and using @extent

Reply:
Fixed @sandeep-deriv , deleted the modal-wrap as we're not using that. Had gotten this css from the previous modal that I was working on. But created an align-items reusable class that I'm calling at modal-icon and at h4.